cuesubmit/cuesubmit/Constants.py

```diff
@@ -38,6 +37,8 @@
 NUKE_RENDER_CMD = config.get('NUKE_RENDER_CMD', 'nuke')
 BLENDER_RENDER_CMD = config.get('BLENDER_RENDER_CMD', 'blender')
 FRAME_TOKEN = config.get('FRAME_TOKEN', '#IFRAME#')
+FRAME_START = config.get('FRAME_START', '#FRAME_START#')
+FRAME_END = config.get('FRAME_END', '#FRAME_END#')
```

Comment:
Let's call these `FRAME_START_TOKEN` and `FRAME_END_TOKEN`, i.e. keep the `_TOKEN` suffix to make it clear what these are for.

Reply:
Resolved in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1337/commits/d3d7c6d89bb1d27d265f879b1f9d51def3b8c1de